packages/xrpl/src/client/index.ts

```diff
@@ -125,9 +125,24 @@ import {
 } from './partialPayment'
 
 export interface ClientOptions extends ConnectionUserOptions {
+  /**
+   * Factor to multiply estimated fee by to provide a cushion in case the
+   * required fee rises during submission of a transaction. Defaults to 1.2.
+   *
+   * @category Fee
+   */
   feeCushion?: number
+  /**
+   * Maximum transaction cost to allow, in decimal XRP. Must be a string-encoded
+   * number. Defaults to '2'.
+   *
+   * @category Fee
+   */
   maxFeeXRP?: string
   proxy?: string
+  /**
+   * Duration to wait for a request to timeout.
+   */
   timeout?: number
```

Comment:
This `timeout` is not used in the codebase. Removal of it didn't cause any failures in the unit or integration tests.

Why is it present?

Reply:
`timeout` is used in `Client.request` on lines 371-374 as a max timeout for a request to be sent, it would only matter if you were in a poor internet connection situation so wouldn't show up easily in the tests. 

Our test coverage isn't good enough to trust passing tests as a sign that a field isn't being used since we're not exercising all code / branches.